Library/Formula/postgresql.rb

```diff
@@ -45,6 +45,10 @@ class Postgresql < Formula
   def install
     ENV.libxml2 if MacOS.version >= :snow_leopard
 
+    ENV.prepend "LDFLAGS", "-L#{Formula['openssl'].opt_lib}"
```

Comment:
hmm, should I use something like %Q(-L#{Formula["openssl"].opt_lib) here or '"\'?